api/src/main/java/org/jmisb/api/klv/st0102/localset/SecurityMetadataLocalSet.java

```diff
@@ -126,6 +128,9 @@ public String displayHeader() {
         return "ST 0102 (local)";
     }
 
+    void getTags() {
```

Comment:
believe this can be removed? since you added getKeys in the base class?

Reply:
Will double check.

Reply:
Should be removed. My apologies for that.